crates/terminal_view/src/terminal_view.rs

```diff
@@ -832,6 +832,17 @@ impl TerminalView {
                 }),
         )
     }
+
+    fn expand_selection_around_cursor(
```

Comment:
1. I don't think we need a method for this, it can be handled at the call site itself.
2. We can also get rid of `cx.notify()`, as it's handled at the call site.